### PR TITLE
Automation of Log_644

### PR DESCRIPTION
**Pull request by gkarager**
Automated test case OCP-32628 - Fluentd continues to ship logs even when one of multiple destination is down
@QiaolingTang @anpingli Please review

### CODE REVIEW

**Comment by gkarager** [comment]
I have run the cases with runner-v3. Here is the pass log link: https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/Runner-v3/305826/console

File: features/logging/cluster_log_forwarder.feature [hunk 1]
```diff
@@ -421,3 +421,83 @@ Feature: cluster log forwarder features
...
+      | infra-container.log |
+    """
+    # delete fluentdserver
+    Given I ensure "fluentdserver" config_map is deleted from the "openshift-logging" project
+    And I ensure "fluentdserver" deployment is deleted from the "openshift-logging" project
+    And I ensure "fluentdserver" service is deleted from the "openshift-logging" project
+    # Again check logs in rsyslogserver
+    Given a pod becomes ready with labels:
+      | appname=rsyslogserver |
+    And I wait up to 300 seconds for the steps to pass:
```
Comment (QiaolingTang) [inline features/logging/cluster_log_forwarder.feature:493]: If the fluentd can't forward logs, the output will still contain  `app-container.log`, `audit.log`, `infra.log` and `infra-container.log`. I don't think it's a valid check.

Comment (gkarager) [reply features/logging/cluster_log_forwarder.feature:493]: Added the check @QiaolingTang

File: features/logging/cluster_log_forwarder.feature [hunk 2]
```diff
@@ -421,3 +421,132 @@ Feature: cluster log forwarder features
...
+    """
+    # check app logs
+    When I execute on the pod:
+      | curl | -sk | -XGET | http://localhost:9200/*/_count?format=JSON | -H | Content-Type: application/json | -d | {"query": {"match": {"kubernetes.namespace_name": "<%= cb.proj1.name %>"}}} |
+    Then the step should succeed
+    And evaluation of `JSON.parse(@result[:stdout])['count']` is stored in the :app_log_count_2 clipboard
+    And the expression should be true> cb.app_log_count_2 > 0
+
+    # check journal logs
```
Comment (anpingli) [inline features/logging/cluster_log_forwarder.feature:532]: The doc count may be changed before before the fluent server is deleted.  So the follow verify steps are invalid

Comment (gkarager) [reply features/logging/cluster_log_forwarder.feature]: @anpingli Removed unnecessary checks, keeping only 'new app' log check.

Comment (anpingli) [reply features/logging/cluster_log_forwarder.feature]: @gkarager  Could you attach the runner-v3 log?

Comment (gkarager) [reply features/logging/cluster_log_forwarder.feature]: @anpingli Here is the link to pass log on runner-v3: https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/Runner-v3/318452/console

**Review by anpingli — COMMENTED** [review]
/lgtm

**Pull request merged**